src/src_V2/modules/manager/financial/components/payables/Payables.module.css

```diff
@@ -1,41 +0,0 @@
-@import '../../../../../styles/colors.css';
-
-.tableContainer {
-  max-height: 680px;
-}
-
-.tableHead,
-.tableRow {
-  height: 56px !important;
-}
-
-/* sticky table head */
-.tableHead {
-  background-color: var(--midGray) !important;
-  position: sticky !important;
-  top: 0;
-  z-index: 1;
-}
-
-.tableBody {
-  font-weight: 400;
-}
-
-.tableHeaderCell {
-  font-weight: 600 !important;
-}
-
-.tableCheckboxCell {
-  padding-left: 3px !important;
-}
-
-.tableCell {
-  width: 140px !important;
-}
-
-.tooltip {
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  width: 100px;
-}
```

Comment:
need we this empty file here?

Reply:
When creating a component, I always create a css file near to it, to show it always has its css module and whenever a style is needed, it must be applied to that css file (not to another file), to keep encapsulation. This habit came from Angular. Just different approaches :D

Reply:
got it, great thanks